src/ng/filter/filters.js

```diff
@@ -699,6 +699,7 @@ function jsonFilter() {
  * @description
  * Converts string to lowercase.
```

Comment:
Please add a link after this that points to uppercase: Format is `{@link ng.filter.uppercase linktext}`